testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeSystemTableCompatibility.java

```diff
@@ -92,4 +92,83 @@ public void testTablePropertiesWithTableFeatures()
             dropDeltaTableWithRetry("default." + tableName);
         }
     }
+
+    @Test(groups = {DELTA_LAKE_OSS, PROFILE_SPECIFIC_TESTS})
+    public void testTablePartitionsWithDeletionVectors()
+    {
+        String tableName = "test_table_partitions_with_deletion_vectors_" + randomNameSuffix();
+        onDelta().executeQuery("" +
+                "CREATE TABLE default." + tableName +
+                "(a INT, b INT, c VARCHAR(1)) " +
+                "USING delta " +
+                "PARTITIONED BY (a, c) " +
+                "LOCATION 's3://" + bucketName + "/databricks-compatibility-test-" + tableName + "' " +
+                "TBLPROPERTIES ('delta.enableDeletionVectors' = true)");
+
+        List<Row> expectedBeforeDelete = ImmutableList.of(
+                row(rowBuilder().addField("a", 1).addField("c", "a").build(), 3L, 1347L,
+                    rowBuilder().addField("b",
+                        rowBuilder().addField("min", 11).addField("max", 19).addField("null_count", 0L).build()).build()),
+                row(rowBuilder().addField("a", 1).addField("c", "d").build(), 1L, 449L,
+                    rowBuilder().addField("b",
+                        rowBuilder().addField("min", 13).addField("max", 13).addField("null_count", 0L).build()).build()));
+
+        List<Row> expectedAfterFirstDelete = ImmutableList.of(
+                row(rowBuilder().addField("a", 1).addField("c", "a").build(), 2L, 898L,
+                    rowBuilder().addField("b",
+                        rowBuilder().addField("min", 11).addField("max", 17).addField("null_count", 0L).build()).build()),
+                row(rowBuilder().addField("a", 1).addField("c", "d").build(), 1L, 449L,
+                    rowBuilder().addField("b",
+                        rowBuilder().addField("min", 13).addField("max", 13).addField("null_count", 0L).build()).build()));
+
+        List<Row> expectedAfterSecondDelete = ImmutableList.of(
+                row(rowBuilder().addField("a", 1).addField("c", "a").build(), 2L, 898L,
+                    rowBuilder().addField("b",
+                        rowBuilder().addField("min", 11).addField("max", 17).addField("null_count", 0L).build()).build()));
```

Comment:
These rows are really hard to read... Could you consider casting to json/varchar type? Same for testTablePartitionsWithNoColumnStats.